dynawo/sources/Models/Modelica/Dynawo/Electrical/Controls/Machines/PowerSystemStabilizers/Standard/BaseClasses/BasePss3.mo

```diff
@@ -63,9 +63,9 @@ model BasePss3 "IEEE power system stabilizer type 3 base model"
     Placement(visible = true, transformation(origin = {-130, 60}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
   Modelica.Blocks.Math.Add add annotation(
     Placement(visible = true, transformation(origin = {-30, 0}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
-  Dynawo.NonElectrical.Blocks.Continuous.TransferFunction transferFunction(a = {A4, A3, 1}, b = {A2, A1, 1}) annotation(
+  Dynawo.NonElectrical.Blocks.Continuous.TransferFunction transferFunction(a = {A4, A3, 1}, b = {A2, A1, 1}, ZeroA1 = true) annotation(
     Placement(visible = true, transformation(origin = {50, 0}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
-  Dynawo.NonElectrical.Blocks.Continuous.TransferFunction transferFunction1(a = {A8, A7, 1}, b = {A6, A5, 1}) annotation(
+  Dynawo.NonElectrical.Blocks.Continuous.TransferFunction transferFunction1(a = {A8, A7, 1}, b = {A6, A5, 1}, ZeroA1 = true) annotation(
```

Comment:
I think with ZeroA1, we should know in advance if the model can have values of 0. As in here it is only possible for Pss3b and Pss3c because we know that we have a test case with values 0. But it is not the case for the other models.
Isn't better to have this modification at the upper level when we will use the model pss3 depending on the application, thus removing the ZeroA1 = true?
I am not sure if it is better to have the default value always equal to true?

Reply:
but I think we should not modify the ZeroA1 here, we should be able to modify it when we define the test case, because it is at that level that we know what values of parameters we are using.
we should have a ZeroA1 parameter for every model with a default value = false

Reply:
I am afraid that a default value ZeroA1 = false in the Modelica model cannot be changed from a .par file, the compilation probably eliminates the relevant "if" branch.

Reply:
I agree but this means that this "if" branch is only useful for the Pss3 models.

For now, maybe you can add this parameter in every model with a default value and for the basePss3 you put a default value = true. 
It is an issue in dynawo that will be solved one day.

Is not adding a default value in the transfer function model and in very model that uses the transfer function and defining it in the .par is also time-consuming?